### PR TITLE
Refactors files_external app commands

### PR DESCRIPTION
**Pull request by fsamapoor**
## Summary
I have made some adjustments to the `apps/files_external/lib/Command` classes to improve code readability.

The improvements in this PR include but are not limited to:

- Using PHP8's constructor property promotion
- Adding return types
- Replacing return code integers with more readable strings.
- Using early returns

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

### CODE REVIEW

**Comment by solracsf** [comment]
CI and Psalm failing 🔴

**Comment by fsamapoor** [comment]
> CI and Psalm failing 🔴

Psalm's not failing. Is it?
It just shows some notices for which I'm not sure if I should make the required adjustment in the scope of this PR.

Other failing actions do not seem to be related to the changes that I have made and none of them are labeled as "Required".
Could you point out any specific failing actions that I can fix?

**Review by come-nc — APPROVED** [review]

File: apps/files_external/lib/Command/Import.php [hunk 1]
```diff
@@ -192,16 +181,16 @@ private function getBackendByClass(string $className) {
...
-	protected function getStorageService($userId) {
-		if (!empty($userId)) {
-			$user = $this->userManager->get($userId);
-			if (is_null($user)) {
-				throw new NoUserException("user $userId not found");
-			}
-			$this->userSession->setUser($user);
-			return $this->userService;
-		} else {
+	protected function getStorageService($userId): StoragesService {
```
[the remark this answers is not in the document]
Comment (fsamapoor) [reply apps/files_external/lib/Command/Import.php]: I also added the type to the other similar method in ListCommand. Thank you.

**Review by artonge — APPROVED** [review]

**Comment by artonge** [comment]
CI failure unrelated

**Pull request merged**